sympy/sets/tests/test_fancysets.py

```diff
@@ -163,7 +166,6 @@ def test_Range_set():
     raises(ValueError, lambda: Range(-oo, oo))
     raises(ValueError, lambda: Range(oo, -oo, -1))
     raises(ValueError, lambda: Range(-oo, oo, 2))
```

Comment:
why are infinities excluded?

Reply:
You can't do anything with an infinite range -- you can't start, end or slice.